content/zh-cn/overview/tasks/observability/tracing/zipkin.md

```diff
@@ -151,4 +151,4 @@ public class KafkaSenderConfiguration {
 
 ### SpringBoot2案例
 
-dubbo-tracing相关的使用在SpringBoot2与3中区别不大，SpringBoot2的案例可参考[代码地址](https://github.com/apache/dubbo-samples/tree/master/4-governance/dubbo-samples-spring-boot-tracing)。
+dubbo-tracing相关的使用在SpringBoot2与3中区别不大，SpringBoot2的案例可参考[代码地址](https://github.com/apache/dubbo-samples/tree/master/4-governance/dubbo-samples-spring-boot3-tracing)。
```

Comment:
@conghuhu PTAL

Reply:
Due to recent adjustments to the dubbo-samples directory related to tracing, I haven't had time to make any changes on the dubbo website yet.
The correct url for SpringBoot2 zipkin sample now should be [https://github.com/apache/dubbo-samples/tree/master/4-governance/dubbo-samples-tracing/dubbo-samples-spring-boot-tracing-zipkin]( https://github.com/apache/dubbo-samples/tree/master/4-governance/dubbo-samples-tracing/dubbo-samples-spring-boot-tracing-zipkin)

Reply:
> Due to recent adjustments to the dubbo-samples directory related to tracing, I haven't had time to make any changes on the dubbo website yet. The correct url for SpringBoot2 zipkin sample now should be https://github.com/apache/dubbo-samples/tree/master/4-governance/dubbo-samples-tracing/dubbo-samples-spring-boot-tracing-zipkin

Cloud you pls submit a PR to fix this page?

Reply:
Okay, I will organize the documents for tracing today

Reply:
thanks！